certbot/src/dns01_client/cloudflare.rs

```diff
@@ -83,7 +83,6 @@ impl Dns01Api for CloudflareClient {
                 "type": "TXT",
                 "name": domain,
                 "content": content,
-                "ttl": 120
```

Comment:
What's the default value if we remove `ttl`?

Reply:
Cloudflare says it is default is 300; however, my test shows it is faster than 120, appearing to be around 60.